### PR TITLE
Add command line interface to negation hierarchy asserter

### PR DESCRIPTION
**Pull request by Shalsh23**

### CODE REVIEW

**Comment by balhoff** [comment]
Sorry, I should have merged this before having you do the other updates to this file. Can you resolve the conflicts?

**Comment by Shalsh23** [comment]
> Sorry, I should have reviewed this before having you do the other updates to this file. Can you resolve the conflicts?

Resolved!

File: src/main/scala/org/phenoscape/owl/NegationHierarchyAsserter.scala [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
+  val factory = OWLManager.getOWLDataFactory
   val Negates = factory.getOWLAnnotationProperty(Vocab.NEGATES)
 
+
+  def main(args: Array[String]): Unit = {
+
+    val manager: OWLOntologyManager = OWLManager.createOWLOntologyManager()
+    val inputOntology: OWLOntology = manager.loadOntologyFromOntologyDocument(new File(args(0)))
+    val axioms: Set[OWLAxiom] = inputOntology.getAxioms(Imports.INCLUDED).asScala.toSet
+    val negationAxioms: Set[OWLAxiom] = assertNegationHierarchy(axioms)
```
[the remark this answers is not in the document]
Comment (Shalsh23) [reply src/main/scala/org/phenoscape/owl/NegationHierarchyAsserter.scala]: Sorry, forgot that. Done that now.

File: src/main/scala/org/phenoscape/owl/NegationHierarchyAsserter.scala [hunk 2]
```diff
@@ -1,26 +1,40 @@
...
+    val negationOntology: OWLOntology = manager.createOntology(negationAxioms.asJava)
+    manager.saveOntology(negationOntology, new FileOutputStream(("negationAxiomsFile")))
```
Comment (balhoff) [inline src/main/scala/org/phenoscape/owl/NegationHierarchyAsserter.scala]: I think you will want to get the filename from `args(1)`.

Comment (Shalsh23) [reply src/main/scala/org/phenoscape/owl/NegationHierarchyAsserter.scala]: Oh, okay.

**Pull request merged**